PRECOMPILATION_GUIDE.md

```diff
@@ -38,7 +38,8 @@ def project do
     make_precompiler: {:nif, CCPrecompiler},
     make_precompiler_url: "https://github.com/cocoa-xu/cc_precompiler_example/releases/download/v#{@version}/@{artefact_filename}",
     make_precompiler_filename: "nif",
-    make_precompiler_priv_paths: ["nif.*"]
+    make_precompiler_priv_paths: ["nif.*"],
+    make_precompiler_unavailable_target: :compile,
```

Comment:
What if we make it a callback in the Precompiler that returns either :compile or :ignore. If no callback is specified, we use `:compile`?

Reply:
I agree! I'll update the code soon. :)